connect/src/wg/cosmo/platform/v1/platform_pb.ts

```diff
@@ -1884,6 +1889,7 @@ export class GetFederatedSubgraphSDLByNameRequest extends Message<GetFederatedSu
   static readonly typeName = "wg.cosmo.platform.v1.GetFederatedSubgraphSDLByNameRequest";
   static readonly fields: FieldList = proto3.util.newFieldList(() => [
     { no: 1, name: "name", kind: "scalar", T: 9 /* ScalarType.STRING */ },
+    { no: 2, name: "federatedGraphName", kind: "scalar", T: 9 /* ScalarType.STRING */ },
```

Comment:
I'd rename the method to `GetLatestValidSubgraphSDLRequest` and revert the naming `latestSdl` and call `lastDeployedSdl` => `updatedAt`. This is simpler.

Reply:
Done